frontend/README.md

```diff
@@ -40,3 +36,18 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
 The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
 
 Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+
+## Linting and Formatting
+
+To use the linter, run:
+
+```
+npm run lint
+```
+
+To use prettier, you can run:
+
+```bash
+npx prettier . --check  # to check what files are already formatted
```

Comment:
Can we add these as scripts that are easier to remember, something akin to `npm run prettier`?